frontend/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Annoying how this gets removed for you but added for me 🤓 , oh well lets ignore it for now.